### PR TITLE
API Doc Generation

### PR DESCRIPTION
**Pull request by athul**
Documentation Generation from `hoppscotch-collection.json`.

New Command: `hopp gen <path to json>`

- [x] Basic FE
- [x] Markdown Support
- [x] Better Styling with docsify

### CODE REVIEW

**Comment by athul** [comment]
Here is a working Demo https://hopp-api1.surge.sh/#/

**Comment by athul** [comment]
hopp-docsify.surge.sh

Final Working URL.

**Review by liyasthomas — APPROVED** [review]
LGTM

**Comment by athul** [comment]
@joicemjoseph Review

**Review by joicemjoseph — COMMENTED** [review]
> @joicemjoseph Review

Will do that in the morning

File: cli.go [hunk 1]
```diff
@@ -7,16 +7,39 @@ import (
...
 	"github.com/fatih/color"
+	"github.com/knadh/stuffbin"
 	"github.com/urfave/cli"
 )
 
 // VERSION is set by `make` during the build to the most recent tag
-var VERSION = ""
+var buildVersion = "unknown"
 
+func initFileSystem(binPath string) (stuffbin.FileSystem, error) {
```
[the remark this answers is not in the document]
Comment (athul) [reply cli.go]: I don't think Exists now. I had removed that in 5f9058c

**Review by joicemjoseph — APPROVED** [review]

**Comment by athul** [comment]
@liyasthomas Merge Cheyyuvaane

**Pull request merged**